src/platforms/hosted/platform.c

```diff
@@ -395,39 +443,42 @@ bool platform_target_get_power(void)
 {
 	switch (info.bmp_type) {
 	case BMP_TYPE_BMP:
-	return remote_target_get_power() ;
+		return remote_target_get_power();
 
 	default:
-		return false ;
+		return false;
 	}
 }
 
 uint32_t platform_target_voltage_sense(void)
 {
-	uint32_t	targetVoltage = 0 ;
+	uint32_t targetVoltage = 0;
+
 	switch (info.bmp_type) {
 	case BMP_TYPE_BMP: {
-		const char * result ;
-		uint32_t units = 0, tenths = 0 ;
-		result = remote_target_voltage() ;
+		const char *result;
+		uint32_t units = 0, tenths = 0;
```

Comment:
These should be moved inside the if, and broken onto separate declaration lines.